### DISCUSSION TITLE
is there a way to proxy to multiple hosts (load balancing) in http_proxy ?

### DISCUSSION THREAD

**Discussion by julien-berthier** [tobychui/zoraxy#229] [General]
Hello,
I would like to know if is there a way to set multiple hosts (for load balancing) on a http_proxy site ?
Have a nice day !

**Comment by tobychui** [comment]
Hi @julien-berthier,

Load balancer are currently wip in Zoraxy version v3.0.8.  It is not available in v3.0.7 (latest release at the time of writing), but I can give you a quick glance on how it will looks like with this screenshot.
![image](https://github.com/tobychui/zoraxy/assets/24617523/8a4642b8-f063-4b75-8902-3623c03860db)

Check back again later :)
